README.md

```diff
@@ -38,8 +25,49 @@ Features
 * Ready to use - just install via composer.json!
 * easy configurable via module config,
 
-How to use `Message` tab?
------------------------------
+Installation
+------------
+
+The preferred way to install this extension is through [composer](http://getcomposer.org/download/).
+
+Either run
+
+```
+php composer.phar require snapshotpl/zf-snap-php-debug-bar "dev-master"
+```
+
+or add
+
+```
+"snapshotpl/zf-snap-php-debug-bar": "dev-master"
+```
+
+to the require section of your `composer.json` file.
```

Comment:
Ok, but you need to rewrite also this line `to the require section of your `composer.json` file.` to `to the require-dev section of your `composer.json` file.`